app/services/user_facing_state.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class UserFacingState
+  attr_reader :document
+  delegate :review_state, :publication_state, to: :document
+
+  def initialize(document)
+    @document = document
+  end
+
+  def to_s
+    if review_state == "submitted_for_review"
```

Comment:
Maybe a case statement or a lookup on a hash of states here would be a bit easier to read? If we match on both review_state and publication_state we're more likely to catch errors where they go out of sync too.

Right now I could imagine a documents review_state being`submitted_for_review` but having an `error_sending_to_draft` publication_state which seems like a state we'd like to raise an exception for.

Reply:
I agree it's a bit iffy now, but I wanted to hold off refactoring until we have the full logic - this is currently missing the scheduling, withdrawal, history mode states & the [error state work](https://trello.com/c/VkVrVGeB). Does that make sense?

Reply:
Makes sense 👍 thanks